docs/irdl.ipynb

```diff
@@ -26,6 +26,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "# Currently no easy way to distribute with xDSL pre-installed.\n",
+    "%pip install xdsl\n",
```

Comment:
Not sure I like this, I would prefer to prompt the user to do it, rather than have this command.
Otherwise, we could also claim that notebooks can be totally moved from the main repo.
Not sure it is a good idea.